src/components/Navigation.jsx

```diff
@@ -2,17 +2,35 @@ import React from 'react';
 import { Link } from 'react-router';
 
 export default class Navigation extends React.Component {
+
   render() {
+
+    // Render all categories in nav bar
+    let renderCategories = () => {
+      let categories = ["opinion", "features", "news"];
+      let data = [];
+
+      categories.forEach(function(category) {
```

Comment:
I'll just use Lodash map again to keep it consistent with the other map functions I've been putting in. Does that sound okay?